### DISCUSSION TITLE
Home Automation

### DISCUSSION THREAD

**Discussion by anonyah** [georgegebbett/recipe-buddy#280] [General]
Hi, 
I am newly using grocy and would love to also use your recipe-buddy development!

My grocy is set up / integrated into Home Automation. Will the recipe-buddy
also work in that surrounding? 

It would be  very nice, if you could give me a short hint.

Thanks!

**Comment by georgegebbett** [comment]
Hi there - 

If you are using Grocy as a Home Assistant add on, then you can use Recipe Buddy with it as long as you have the Grocy API exposed externally. 

Recipe Buddy is not distributed as a Home Assistant addon, and will need to be run separately, as per the instructions in the README
